util/dvsim/dvsim.py

```diff
@@ -168,8 +168,13 @@ def resolve_proj_root(args):
     # Check if jobs are dispatched to external compute machines. If yes,
     # then the repo needs to be copied over to the scratch area
     # accessible to those machines.
+    # If --purge env var is set, purge the scratch area repo_top before
+    # copying over.
     if args.remote:
         dest = os.path.join(args.scratch_root, args.branch, "repo_top")
+        if args.purge:
+            log.info("Purging remote top_repo path %s", dest)
+            os.system("/bin/rm -rf " + dest)
```

Comment:
I think `shutil.rmtree()` probably works better here.

Reply:
Oh sure, I will change it to use shutil!